internal/pkg/config/v1/api_key_pair.go

```diff
@@ -1,7 +1,28 @@
 package v1
 
+import (
+	"runtime"
+	"strings"
+
+	"github.com/confluentinc/cli/internal/pkg/secret"
+)
+
 // APIKeyPair holds an API Key and Secret.
 type APIKeyPair struct {
 	Key    string `json:"api_key"`
 	Secret string `json:"api_secret"`
```

Comment:
These should probably be `omitempty` so the configuration file looks a little cleaner.